src/ipc/ipc4/dai.c

```diff
@@ -28,6 +27,7 @@
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
+#include <../audio/copier/dai_copier.h>
```

Comment:
Please use "../audio/copier" so this is relative to this source file.

Reply:
I am still using: target_include_directories(sof_options INTERFACE ${RIMAGE_TOP}/src/include)
othewise, it should be:  "../../audio/copier"

these include path come from:
target_include_directories(SOF INTERFACE ${RIMAGE_TOP}/src/include)
target_include_directories(SOF INTERFACE ${SOF_SRC_PATH}/include)
in zephyr/CMakeList.txt.

Reply:
This is good as its gives us more visibility on where we have stickyness around legacy abstractions.

Reply:
@btian1 @lgirdwood I'd still vote for "#include "../audio/copier" . and not "#include <.../audio/...". Mixing relative include paths and system includes is not clean -> https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html